xyz-models/src/main/java/com/here/xyz/models/geojson/implementation/Geometry.java

```diff
@@ -189,8 +191,17 @@ public static void validateLinearRingCoordinates(final Object raw, boolean isExt
     // reject Polygons that do not follow the right-hand rule.
     //
     // --> Therefore we will not check the right-hand rule here, even while it would be possible!
+    HashSet<String> hset = new HashSet<String>();
+
     for (int i = 0; i < points.size(); i++) {
       final Object point = points.get(i);
+      String strPoint = point.toString();
```

Comment:
Serializing all the points can be quite a heavy load for large geometries. I guess we could add some Point#equals() and/or Point#hashCode() method for better efficiency.